chapter_multilayer-perceptrons/numerical-stability-and-init_vn.md

```diff
@@ -79,7 +79,10 @@ Since early artificial neural networks were inspired by biological neural networ
 Let's take a closer look at the function to see why picking it might be problematic vis-a-vis vanishing gradients.
 -->
 
-*dịch đoạn phía trên*
+Một thủ phạm chính gây ra vấn đề tiêu biến gradient là hàm kích hoạt $\sigma$ được chọn để đặt xen giữa các toán hạng tuyến tính tại mỗi tầng.
```

Comment:
mình tra `toán hạng` thì có nghĩa giống `input`, `operation` là `phép toán` thì đúng hơn
```suggestion
Một thủ phạm chính gây ra vấn đề tiêu biến gradient là hàm kích hoạt $\sigma$ được chọn để đặt xen giữa các phép toán tuyến tính tại mỗi tầng.
```

Reply:
`operator` cũng có thể dịch là `toán tử ` (https://vi.wikipedia.org/wiki/To%C3%A1n_t%E1%BB%AD)